CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.1.2 (2019-12-30)
+
+- Fixed the wait component to prevent throwing errors when using the keyboard. [#50](https://github.com/blackbaud/skyux-indicators/pull/50)
```

Comment:
Is `wait component` correct? In the linked PR, it looks like the changes are to the tokens component (`tokens.component.ts`) and a wait service (`wait-adapter.service.ts`).

Also, do we need "throwing"? I think this means the same thing without that word and that it would read better without it.

Reply:
That works for me. Thanks for catching the tokens component - I missed that. I've updated this based on your feedback.